sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -325,6 +347,8 @@ class SQLQueryTestSuite extends QueryTest with SharedSparkSession {
         localSparkSession.conf.set(SQLConf.CROSS_JOINS_ENABLED.key, true)
         localSparkSession.conf.set(SQLConf.ANSI_ENABLED.key, true)
```

Comment:
We should remove it in the followup, and see if it fails tests. Ideally pgsql dialect should not be affected by ansi mode config. cc @gengliangwang